src/js/RacingCar.js

```diff
@@ -0,0 +1,28 @@
+import { $ } from './dom.js';
+import NameContainer from './container/NameContainer.js';
+import NumberContainer from './container/NumberContainer.js';
+import RacingCarContainer from './container/RacingCarContainer.js';
+import store from './store/store.js';
+
+class RacingCar {
+  constructor() {
+    this.$carName = $('.car-name');
+    this.$carRacingNumber = $('.car-racing-number');
+    this.$result = $('.result');
+
+    this.render();
+  }
+
+  render() {
+    const { cars, racingNumber } = store.getState();
+    NameContainer(this.$carName);
+
+    if (cars.length === 0) return;
```

Comment:
```suggestion
    if (!cars.length) return;
```

축약해서 작성해도 좋을거 같네용(취향에 따라 갈릴 수 있는 부분이라 적용하시라는 얘기는 아닙니다!)